Mathlib/AlgebraicGeometry/Scheme.lean

```diff
@@ -215,52 +285,43 @@ theorem inv_val_c_app {X Y : Scheme} (f : X ⟶ Y) [IsIso f] (U : Opens X) :
 theorem inv_val_c_app_top {X Y : Scheme} (f : X ⟶ Y) [IsIso f] :
     (inv f).val.c.app (op ⊤) = inv (f.val.c.app (op ⊤)) := by simp
 
-/-- Given a morphism of schemes `f : X ⟶ Y`, and open sets `U ⊆ Y`, `V ⊆ f ⁻¹' U`,
-this is the induced map `Γ(Y, U) ⟶ Γ(X, V)`. -/
-abbrev Hom.appLe {X Y : Scheme} (f : X ⟶ Y) {V : Opens X} {U : Opens Y}
-    (e : V ≤ f ⁻¹ᵁ U) : Γ(Y, U) ⟶ Γ(X, V) :=
-  f.1.c.app (op U) ≫ X.presheaf.map (homOfLE e).op
-#align algebraic_geometry.Scheme.hom.app_le AlgebraicGeometry.Scheme.Hom.appLe
+end Scheme
 
 /-- The spectrum of a commutative ring, as a scheme.
 -/
 def specObj (R : CommRingCat) : Scheme where
   local_affine _ := ⟨⟨⊤, trivial⟩, R, ⟨(Spec.toLocallyRingedSpace.obj (op R)).restrictTopIso⟩⟩
   toLocallyRingedSpace := Spec.locallyRingedSpaceObj R
-#align algebraic_geometry.Scheme.Spec_obj AlgebraicGeometry.Scheme.specObj
+#align algebraic_geometry.Scheme.Spec_obj AlgebraicGeometry.specObj
 
-@[simp]
-theorem specObj_toLocallyRingedSpace (R : CommRingCat) :
+theorem Spec_toLocallyRingedSpace (R : CommRingCat) :
     (specObj R).toLocallyRingedSpace = Spec.locallyRingedSpaceObj R :=
   rfl
-#align algebraic_geometry.Scheme.Spec_obj_to_LocallyRingedSpace AlgebraicGeometry.Scheme.specObj_toLocallyRingedSpace
+#align algebraic_geometry.Scheme.Spec_obj_to_LocallyRingedSpace AlgebraicGeometry.Spec_toLocallyRingedSpace
 
 /-- The induced map of a ring homomorphism on the ring spectra, as a morphism of schemes.
 -/
-def specMap {R S : CommRingCat} (f : R ⟶ S) : specObj S ⟶ specObj R :=
+def SpecMap {R S : CommRingCat} (f : R ⟶ S) : specObj S ⟶ specObj R :=
```

Comment:
This seems dubious.

Reply:
Do you mean misleading?
I believe the naming convention allow names that are capitalized in the literature as exceptions?
Or else `Scheme.Spec` should be `Scheme.spec` according to the convention.

Reply:
I've removed the renaming from this PR.